docs/whats_new.rst

```diff
@@ -1184,4 +1194,4 @@ Bug Fixes
 - Mentions are now triggered normally. This was changed due to the way discord handles it internally.
 - Fix issue when a :class:`Message` would attempt to upgrade a :attr:`Message.server` when the channel is
   a :class:`Object`.
-- Unavailable servers were not being added into cache, this has been corrected.
\ No newline at end of file
+- Unavailable servers were not being added into cache, this has been corrected.
```

Comment:
Why did this change?

Reply:
it just reflects the amount of lines that changed by inserting the above part